organization.go

```diff
@@ -158,6 +159,9 @@ type OrganizationCreateOptions struct {
 
 	// Optional: SendPassingStatusesForUntriggeredSpeculativePlans toggles behavior of untriggered speculative plans to send status updates to version control systems like GitHub.
 	SendPassingStatusesForUntriggeredSpeculativePlans *bool `jsonapi:"attr,send-passing-statuses-for-untriggered-speculative-plans,omitempty"`
+
+	// Optional: AssessmentsEnforced toggles whether health assessment enablement is enforced across all assessable workspaces or if the decission to enabled health assessments is delegated to the workspace setting AssessmentsEnabled.
```

Comment:
```suggestion
	// Optional: AssessmentsEnforced toggles whether health assessment enablement is enforced across all assessable workspaces or if the decision to enabled health assessments is delegated to the workspace setting AssessmentsEnabled.
```

Could you clarify what an assessable workspace is?